public/modules/custom/helfi_rekry_content/helfi_rekry_content.module

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+
+use Drupal\migrate\MigrateSkipRowException;
+
+/**
+ * Format filename from source uri
+ * 
+ * @param string|null
+ *   The source uri
+ * 
+ * @return string|null
+ *   Filename or null
+ */
+function _helfi_rekry_content_filename(string|null $sourceUri = NULL): ?string  {
+  if(!$sourceUri) {
```

Comment:
Looks like module file is not run through phpcs for some reason. There should be a space between `if `  and `(`.